tests/func-tests/node_placement_test.go

```diff
@@ -36,28 +36,27 @@ var _ = Describe("[rfe_id:4356][crit:medium][vendor:cnv-qe@redhat.com][level:sys
 		originalWorkloadSpec hcov1beta1.HyperConvergedConfig
 		cli                  client.Client
 		cliSet               *kubernetes.Clientset
+		workerNodes          *v1.NodeList
 	)
 
 	BeforeAll(func(ctx context.Context) {
 		cli = tests.GetControllerRuntimeClient()
 		cliSet = tests.GetK8sClientSet()
 
-		nodes := listNodesByLabels(ctx, cliSet, "node-role.kubernetes.io/control-plane!=")
-		tests.FailIfSingleNode(len(nodes.Items) < 2)
+		workerNodes = listNodesByLabels(ctx, cliSet, "node-role.kubernetes.io/worker")
+		tests.FailIfSingleNode(len(workerNodes.Items) < 2)
 
-		// Label all but first node with "node.kubernetes.io/hco-test-node-type=infra"
-		// We are doing this to remove dependency of this Describe block on a shell script that
-		// labels the nodes this way
+		// Label all but the last node with "node.kubernetes.io/hco-test-node-type=infra"
```

Comment:
now it's "all but the first node", no? 🙂

Reply:
yes. sorry. fixed

Reply:
It took me some time to understand that the error is in the comment and not in the code. leftover...